zapcore/level.go

```diff
@@ -55,6 +55,21 @@ const (
 	_maxLevel = FatalLevel
 )
 
+// UnmarshalLevel unmarshals a level based on the lower-case or all-caps ASCII
+// representation of the log level. If the provided ASCII representation is
+// invalid an error is returned.
+//
+// This is particularly useful when dealing with text input to configure log
+// levels.
+func UnmarshalLevel(text string) (Level, error) {
```

Comment:
I know I suggested Unmarshal but I realize now that Parse might be more appropriate.
Similar to time.Parse, for example.

```suggestion
// ParseLevel parses a level based on the lower-case or all-caps ASCII
// representation of the log level. If the provided ASCII representation is
// invalid an error is returned.
//
// This is particularly useful when dealing with text input to configure log
// levels.
func ParseLevel(text string) (Level, error) {
```